lib/irule-tester

```diff
@@ -323,12 +374,12 @@ function should_select_pool {
   local reqUrl=$2
   local destPool=$3
 
-  local poolResp=$($curlCmd -A "$userAgent" $reqUrl -I | egrep -o "^Set-Cookie: BIGipServer${destPool}|Set-Cookie: LastSelectedPool=/[A-Za-z0-9_-]*/${destPool}" | dos2unix)
```

Comment:
I wanted to have the test tool return the name of the pool that was actually selected when the expected pool was not found.

Jon Abend
C (312) 446-3914

From: Jesse Mauntel [mailto:notifications@github.com]
Sent: Monday, March 09, 2015 10:20 AM
To: jmauntel/irule-tester
Cc: Abend, Jon
Subject: Re: [irule-tester] Added two new functions and updated two existing to extend framework (#1)


In lib/irule-tester<https://github.com/jmauntel/irule-tester/pull/1#discussion_r26044198>:

> @@ -323,12 +374,12 @@ function should_select_pool {

>    local reqUrl=$2

>    local destPool=$3

>

> -  local poolResp=$($curlCmd -A "$userAgent" $reqUrl -I | egrep -o "^Set-Cookie: BIGipServer${destPool}|Set-Cookie: LastSelectedPool=/[A-Za-z0-9_-]*/${destPool}" | dos2unix)

Can you help understand what the significance of this block of change is? I'm not seeing why this is better.

—
Reply to this email directly or view it on GitHub<https://github.com/jmauntel/irule-tester/pull/1/files#r26044198>.